packages/plugin-core/src/commands/RefactorHierarchyV2.ts

```diff
@@ -32,27 +55,119 @@ export class RefactorHierarchyCommandV2 extends BasicCommand<
   CommandOutput
 > {
   key = DENDRON_COMMANDS.REFACTOR_HIERARCHY.key;
-  async gatherInputs(): Promise<CommandOpts | undefined> {
+
+  async promptScope(opts: {
+    initialValue: string;
+  }): Promise<NoteLookupProviderSuccessResp | undefined> {
+    const { initialValue } = opts;
+    const lcOpts: LookupControllerV3CreateOpts = {
+      nodeType: "note",
+      disableVaultSelection: true,
+      vaultSelectCanToggle: false,
+      extraButtons: [
+        Selection2ItemsBtn.create(false),
+        MultiSelectBtn.create(false),
+      ],
+    };
+    const controller = LookupControllerV3.create(lcOpts);
+    const provider = new NoteLookupProvider(this.key, {
+      allowNewNote: false,
+      noHidePickerOnAccept: false,
+    });
+    return new Promise((resolve) => {
+      NoteLookupProviderUtils.subscribe({
+        id: this.key,
+        controller,
+        logger: this.L,
+        onDone: (event: HistoryEvent) => {
+          const data = event.data as NoteLookupProviderSuccessResp;
+          if (data.cancel) {
+            resolve(undefined);
+          }
+          resolve(data);
+        },
+        onHide: () => {
+          resolve(undefined);
+        },
+      });
+      controller.show({
+        initialValue,
+        title: "Scope",
+        placeholder: "Query for scope. Press esc. to use all notes.",
+        provider,
+      });
+    });
+  }
+
+  async promptMatchText() {
+    const match = await VSCodeUtils.showInputBox({
+      prompt: "Enter match text. Leave blank to capture entire file name.",
```

Comment:
I know this is parity with our current behavior, but should we add some more info here as to what 'match text' does? We can also split it up to use the 'title' prop and the 'prompt' prop of `InputBox`.  Maybe something like 

Title: "Enter match text"

Prompt: "The matched portion of the file name will be the part that gets modified. The rest will remain unchanged.  Leave blank to capture entire file name"